### DISCUSSION TITLE
Inertia Feature: Suggestions for this coming Inertia 2.0 Version

### DISCUSSION THREAD

**Discussion by imnash21** [inertiajs/inertia#1951] [Ideas]
Is this possible that this coming v2.0 will have new features for page handling like Laravel Folio, Nuxt, and Next that can automatically become a page for all files inside the pages directory/folder?

**Comment by reinink** [comment]
So this is already possible with just a little bit of work in your app — for example @aarondfrancis has a blog post on this:

https://aaronfrancis.com/2024/file-based-routing-in-inertiajs-c40ed2db

We actually do this on the inertiajs.com website as well:

https://github.com/inertiajs/inertiajs.com/blob/f685a1d8be00425fd4b535fb20a26aac0517d313/routes/web.php#L18-L24

**Reply by imnash21** [reply]
Wow great, working as expected, I love it, thank you so much sir Mr. Reinink @reinink.

Is this possible to be a part of Inertia 2.0, a built-in function/method?

Route::inertiaPages(...);
